src/main/java/basic/Exercise20.java

```diff
@@ -0,0 +1,21 @@
+package basic;
+
+public class Exercise20 {
+
+    public boolean checkHappyNum(int num) {
+
+        int sum = 0;
+        do {
+            int mod = num % 10;
+            sum += mod * mod;
+            num /= 10;
+            if (num == 0) {
+                if (sum / 10 == 0) break;
+                num = sum;
+                sum = 0;
+            }
+        } while (num != 0);
+
+        return sum == 1 ? true : false;
```

Comment:
Dòng này tương đương với
```suggestion
        return sum == 1 ;
```

Reply:
Done